usr/src/cmd/bhyve/bhyverun.c

```diff
@@ -110,6 +110,9 @@ __FBSDID("$FreeBSD$");
 #include "rtc.h"
 #include "vga.h"
 #include "vmgenc.h"
+#ifndef __FreeBSD__
```

Comment:
should't this reather be a positive condition :) or is the privileges.h inherently portable, just not to FreeBDS ?

Reply:
This is how all of the illumos divergence from the upstream freebsd bhyve is done in illumos-gate - bracketed with __FreeBSD__